### DISCUSSION TITLE
php 8.1.4 fails post-install on M1 b/c pear command terminated by SIGKILL

### DISCUSSION THREAD

**Discussion by mbirth** [Homebrew/discussions#3095] [Everyday usage] [answered]
Ran into #96892 but also when installing php 8.1.4, I get:

    Warning: The post-install step did not complete successfully
    You can try again using:
      brew postinstall php
    ==> An exception occurred within a child process:
      BuildError: Failed executing: /opt/homebrew/Cellar/php/8.1.4/bin/pear config-set php_ini /opt/homebrew/etc/php/8.1/php.ini system

Trying to run pear manually gives:

    fish: Job 1, 'pear' terminated by signal SIGKILL (Forced quit)

Can anyone confirm?

EDIT: Console shows "Termination Reason:    Namespace CODESIGNING, Code 2" and I've noticed that ffmpeg also gets killed now due to the same reason.

**Comment by carlocab** [comment, marked as answer]
Try
```
brew reinstall $(brew deps php) $(brew deps ffmpeg) php ffmpeg
```

**Reply by mbirth** [reply]
Thanks, the complete reinstall seemed to have done the trick.
